### PR TITLE
Reformat daily digest text email

### PR DESCRIPTION
**Pull request by kenyonj**
Before:

After:

### CODE REVIEW

File: web/controllers/email_preview_controller.ex [hunk 1]
```diff
@@ -9,11 +9,11 @@ defmodule Constable.EmailPreviewController do
       email_body(email_name) |> Repo.rollback
     end
 
-    html(conn, body)
+    text(conn, body)
   end
 
   defp email_body(email_name) do
-    email_for(String.to_atom(email_name)).html_body
+    email_for(String.to_atom(email_name)).text_body
```
Comment (SViccari) [inline web/controllers/email_preview_controller.ex]: Was this intended to be a temporary change? Or do you prefer for the email preview to display the text version vs the html version?

File: web/templates/email/daily_digest.text.eex [hunk 2]
```diff
@@ -1,31 +1,33 @@
...
 <% end %>
 <%= unless Enum.empty?(@announcements) do %>
-<%= gettext("Recently added announcements") %>
+<%= gettext("Recently added announcements\n") %>
 -----------------
 
 <%= for announcement <- @announcements do %>
   <%= announcement.title %> - <%= announcement_url(Constable.Endpoint, :show, announcement) %>
   <%= gettext("posted by") %> <%= announcement.user.name %> <%= gettext("in") %> <%= interest_names(announcement) %>
+  <%= gettext("\n") %>
```
Comment (SViccari) [inline web/templates/email/daily_digest.text.eex]: Perhaps `<%= "\r\n" %>` would work here as well, and below on [line 30](https://github.com/thoughtbot/constable/pull/429/files#diff-bd64116ecaddf010b9c23b36a38cdee3R30)?

File: web/controllers/email_preview_controller.ex (hunk 1 above)
Comment (kenyonj) [inline web/controllers/email_preview_controller.ex]: Yes, this was temporary, thanks for catching that! Fixed in 693eb62

File: web/templates/email/daily_digest.text.eex [hunk 3]
```diff
@@ -1,31 +1,33 @@
...
 <% end %>
 <%= unless Enum.empty?(@comments) do %>
-<%= ngettext("%{count} announcement was discussed", "%{count} announcements were discussed", length(discussed_announcements(@comments))) %>
+<%= ngettext("%{count} announcement was discussed ", "%{count} announcements were discussed ", length(discussed_announcements(@comments))) %>
 <%= ngettext("by %{count} person", "by %{count} people", length(unique_commenters(@comments))) %>
 
 <%= for announcement <- discussed_announcements(@comments) do %>
   <%= announcement.title %> - <%= announcement_url(Constable.Endpoint, :show, announcement) %>
   <%= new_comment_count_text(@comments, announcement) %> <%= commenters_list_text(announcement, @comments) %>
+  <%= gettext("\n") %>
```
Comment (paulcsmith) [inline web/templates/email/daily_digest.text.eex]: Same here. I don't think `gettext` is needed

**Comment by paulcsmith** [comment]
Thanks for the review @SViccari, and nice find @kenyonj. LGTM!

File: web/templates/email/daily_digest.text.eex (hunk 2 above)
Comment (kenyonj) [inline web/templates/email/daily_digest.text.eex]: agreed, thanks @SViccari and @paulcsmith ... I think we can get away with just the newline character `\n` without the need for the carriage return character `\r`.

Fixed in 0ebf7f3

**Comment by kenyonj** [comment]
Thanks for the review @SViccari and @paulcsmith

**Pull request merged**